### PR TITLE
Add missing dependency image_geometry

### PR DESCRIPTION
**Pull request by mcopejans**
@PerrineAguiar PTAL

### CODE REVIEW

File: tango_ros_common/tango_ros_native/package.xml [hunk 1]
```diff
@@ -12,6 +12,7 @@
   <build_depend>cmake_modules</build_depend>
   <build_depend>dynamic_reconfigure</build_depend>
   <build_depend>glog_catkin</build_depend>
+  <build_depend>image_geometry</build_depend>
```
Comment (PerrineAguiar) [inline tango_ros_common/tango_ros_native/package.xml]: I think you also need to add this package in the CMakeLists file. Inside `find_package` and `catkin_package`

**Comment by mcopejans** [comment]
The pinhole_camera model ([link](https://github.com/Intermodalics/tango_ros/blob/424171cb9d0cb53e50800fd139557323363c538b/tango_ros_common/tango_ros_native/include/tango_ros_native/tango_ros_node.h#L31)) depends on it.

**Comment by mcopejans** [comment]
@PerrineAguiar Thanks for your review. I addressed your comments.

**Comment by PerrineAguiar** [comment]
LGTM, thanks for the quick answer.

**Pull request merged**